website/docs/cloud-docs/integrations/kubernetes/api-reference.mdx

```diff
@@ -142,74 +167,86 @@ _Appears in:_
 
 
 
-ConsumerWorkspace allows access to the state for specific workspaces within the same organization. Only one of the fields `ID` or `Name` is allowed. At least one of the fields `ID` or `Name` is mandatory. [More information](/terraform/cloud-docs/workspaces/state#remote-state-access-controls)
+ConsumerWorkspace allows access to the state for specific workspaces within the same organization.
+Only one of the fields `ID` or `Name` is allowed.
+At least one of the fields `ID` or `Name` is mandatory.
+More information:
+  - https://developer.hashicorp.com/terraform/cloud-docs/workspaces/state#remote-state-access-controls
```

Comment:
is there a reason why we put these on three different lines? it renders the same way

<img width="728" alt="image" src="https://github.com/user-attachments/assets/347b31a4-8a98-4594-89d4-2ece3461e400">

Reply:
Updated this across the doc